### ISSUE TITLE
Import geojson to mark the Priority area

### ISSUE THREAD

**Issue by abhisheksaikia** [hotosm/osm-tasking-manager2#809]
At present, tasking manager accepts a geojson file for creating the task, but not for defining the priority area which requires drawing manually. An option to import a geojson to define the priority areas will be useful and convenient.

**Comment by maning** [comment]
This a great feature, one thing we need to consider though is that in some cases especially during HOT activation, priority areas change as we evaluate the conditions on the ground.  I see 2 options:

* Imported geojson for priority areas is not editable (make this explicit in the help).
* Allow editing of imported geojson.

**Comment by dalekunce** [comment]
I agree with @maning having the editable priority area is a requirement. We will often move the priority area during an activation, or even a mapathon, to keep people focused to make sure we are creating good map data the whole time.
